docs/source/data-section.rst

```diff
@@ -94,7 +94,7 @@ This is represented in the following way:
 Note that the actual mnemonic is not present, to avoid ambiguity about
 which curve would be expected to be returned:
 
-.. code-block:: python
+.. code-block::
```

Comment:
sphinx was complaining about this not being python code so I remove 'python'.

Reply:
Note: I changed this in the back to `.. code-block:: python` and added a second block:  `.. code-block:: console` for the output.  This enables the python code to be highlight but separates it slightly from the output.

There should be a better way to do this but this works for now.